build/azure-pipelines.pr-automatic.yml

```diff
@@ -75,3 +75,45 @@ jobs:
       make build-bundle validate-bundle
     workingDirectory: '$(System.DefaultWorkingDirectory)'
     displayName: 'Validate Example Bundles'
+
+- job: integration_test
+  dependsOn: setup
+  condition: eq(dependencies.setup.outputs['BUILD.DOCS_ONLY'], 'false')
+  steps:
+  - task: GoTool@0
+    inputs:
+      version: '$(GOVERSION)'
+  - script: build/azure-pipelines.setup-go-workspace.sh
+    displayName: 'Set up the Go workspace'
+
+  - script: |
+      make build
+    workingDirectory: '$(System.DefaultWorkingDirectory)'
+    displayName: 'Build'
+
+  - script: |
+      trap 'make -f Makefile.kind delete-kind-cluster' EXIT
```

Comment:
Oh! I just realized one thing, sorry I missed it before. These scripts can't have multiple steps in them unless you either you `set -e` at the top, to have it stop on first error, or what I recommend is to have any steps be a single script that you call.

Can you follow-up in another PR and roll this into a single makefile/script so that we don't have to worry about this problem?